Library/Formula/pandoc.rb

```diff
@@ -24,7 +24,12 @@ class Pandoc < Formula
 
   def install
     cabal_sandbox do
-      cabal_install "--only-dependencies"
+      args = []
+      if MacOS.version <= :lion
+        # disable AESNI for OSX <= 10.7
+        args << "--constraint='cryptonite -support_aesni'"
```

Comment:
That was it, now working. Thanks a lot!

How would we optionally build with a current GCC? With the new check, even `--cc=gcc-5` would not use AESNI. Is there a way to catch the cc flag if passed and then see if the version is good enough?